socorro/unittest/external/es/test_supersearch.py

```diff
@@ -426,6 +426,65 @@ def test_get_with_bool_operators(self):
         ok_(not res['hits'][0]['accessibility'])
 
     @minimum_es_version('1.0')
+    def test_get_with_combined_operators(self):
+        self.index_crash({
+            'signature': 'js::break_your_browser',
+            'app_notes': 'foo bar mozilla',
+            'product': 'WaterWolf',
+            'date_processed': self.now,
+        })
+        self.index_crash({
+            'signature': 'mozilla::js::function',
+            'app_notes': 'foo bar',
+            'product': 'WaterWolf',
+            'date_processed': self.now,
+        })
+        self.index_crash({
+            'signature': 'js<isKewl>',
+            'app_notes': 'foo mozilla',
+            'product': 'EarthRacoon',
+            'date_processed': self.now,
+        })
+        self.index_crash({
+            'signature': 'foo(bar)',
+            'app_notes': 'mozilla bar',
+            'product': 'EarthRacoon',
+            'date_processed': self.now,
+        })
+        self.refresh_index()
+
+        res = self.api.get(
+            signature=['js', '~::'],
+        )
+
+        eq_(res['total'], 3)
+        eq_(len(res['hits']), 3)
```

Comment:
I think you should do something like...:
```python
eq_([x['signature'] for x in res['hits']], [sig1, sig2, sig3])
```
That way the test gets a lot easier to read.